openslides/utils/autoupdate.py

```diff
@@ -71,41 +69,47 @@ def on_close(self):
 
     def handle_rest_request(self, response):
         """
-        Handler that is called when the rest api responds.
+        Sends data to the client of the connection instance.
 
-        Sends the response.body to the client.
+        This method is called after succesful response of AsyncHTTPClient().
+        See send_object().
         """
-        # TODO: update cookies
+        # TODO: Update cookies of the client.
+        url_parts = response.request.url.split('/')
+        data = {
+            'url': response.request.url,
+            'name': '%s/%s' % (url_parts[-4], url_parts[-3]),
+            'data': json.loads(response.body.decode()),}
```

Comment:
could you write a comment that explains what parts of the url is inside url_parts[-3] and url_parts[-4]?

I can live with this solution but I am not happy. I have some concerns that this will work in any place. For example, what is with the projector? Could it be possible, that we will push a list of models in the future?